### DISCUSSION TITLE
Elixir SDK

### DISCUSSION THREAD

**Discussion by nileshtrivedi** [zitadel/zitadel#8199] [Ideas]
It would be helpful to have an SDK for Elixir language and the Phoenix Framework

Here are a couple of auth libraries in Elixir which support multiple strategies where Zitadel support could be added:
- Assent: https://github.com/pow-auth/assent
- UeberAuth: https://github.com/ueberauth/ueberauth

**Comment by nileshtrivedi** [comment]
I stumbled upon this repo: https://github.com/maennchen/zitadel_api

If this works well, I'd recommend mentioning this on Zitadel docs site.

**Reply by fforootd** [reply]
I think we already link to it here https://zitadel.com/docs/sdk-examples/introduction

**Reply by nileshtrivedi** [reply]
Oh nice, I had looked at that page but missed it somehow while scanning. Thanks. 

I think this discussion should still be kept open for making this library officially supported and perhaps building connectors for the auth frameworks I mentioned. I have submitted [a feature request](https://github.com/maennchen/zitadel_api/issues/6) in that repo.

**Reply by fforootd** [reply]
Sure let's keep this here for people to find 👍
